latest/docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -95,6 +95,33 @@ az ad sp create-for-rbac --name {appId} --password "{strong password}"
  > Don't create an insecure password.  Follow the
  > [Azure AD password rules and restrictions](/azure/active-directory/active-directory-passwords-policy) guidance.
 
+### Create the service principal with self-signed certificate
+
+Use [az ad sp create-for-rbac](/cli/azure/ad/sp#create-for-rbac) and the `--create-cert` parameter to create a self-signed certificate.
+
+```azurecli-interactive
+az ad sp create-for-rbac --name {appId} --create-cert
+```
+
+It returns the message: `Please copy <path>/<file-name>.pem to a safe place. When run 'az login' provide the file path to the --password argument`
```

Comment:
If the important information is contained in the message, why not only print the message? Is the message also displayed in different output modes? If it is, this could cause problems with automation via CLI, which relies on output being JSON or TSV parsable.

Recommend removing this and leaving the output only.